x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/exporter/local/LocalExporter.java

```diff
@@ -237,13 +232,12 @@ LocalBulk resolveBulk(ClusterState clusterState, boolean clusterStateChange) {
      * monitoring cluster (this one, as the local exporter) is not setup yet.
      *
      * @param clusterState The current cluster state.
-     * @param templates All template names that should exist.
      * @return {@code true} indicates that all resources are available and the exporter can be used. {@code false} to stop and wait.
      */
-    private boolean setupIfNotElectedMaster(final ClusterState clusterState, final Set<String> templates) {
+    private boolean setupIfNotElectedMaster(final ClusterState clusterState) {
         // any required template is not yet installed in the given cluster state, we'll wait.
-        for (final String template : templates) {
-            if (hasTemplate(clusterState, template) == false) {
+        for (final String template : MonitoringTemplateUtils.TEMPLATE_IDS) {
+            if (hasTemplate(clusterState, MonitoringTemplateUtils.templateName(template)) == false) {
```

Comment:
Should you use the static `templateName` import here and throughout? It looks like you might be mix 'n matching.

Reply:
Ah, this is a backport PR -- nevermind me!